pulpcore/cli/core/exporter.py

```diff
@@ -101,17 +101,17 @@ def update(
     repository: Iterable[EntityFieldDefinition],
     repository_href: Iterable[str],
 ) -> None:
-    the_exporter = exporter_ctx.entity
     exporter_href = exporter_ctx.pulp_href
+    payload: Dict[str, Any] = {}
 
     if path:
-        the_exporter["path"] = path
+        payload["path"] = path
 
     if repository or repository_href:
-        the_exporter["repositories"] = [
+        payload["repositories"] = [
             repository_ctx.pulp_href
             for repository_ctx in repository
             if isinstance(repository_ctx, PulpEntityContext)
         ] + list(repository_href)
 
-    exporter_ctx.update(exporter_href, the_exporter)
+    exporter_ctx.update(exporter_href, payload)
```

Comment:
Is there a reason for this change to be linked to the openapi-validation?

Reply:
Glad that you ask! Yes. `the_exporter` is the answer of a `GET` request and it contains fields like `pulp_created`, `pulp_href` and so on. Those are not meant to be in the set of parameters of a `PATCH` call. And out new linting alerts on it.